runtime/src/main/java/org/corfudb/runtime/CheckpointLivenessUpdater.java

```diff
@@ -0,0 +1,67 @@
+package org.corfudb.runtime;
+
+import com.google.protobuf.Message;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.collections.CorfuStore;
+import org.corfudb.runtime.collections.Table;
+import org.corfudb.runtime.collections.TableOptions;
+import org.corfudb.runtime.collections.TxnContext;
+import org.corfudb.runtime.CorfuStoreMetadata.TableName;
+import org.corfudb.runtime.object.transactions.TransactionalContext;
+import org.corfudb.runtime.proto.service.CorfuMessage;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import static org.corfudb.runtime.view.TableRegistry.CORFU_SYSTEM_NAMESPACE;
+
+@Slf4j
+public class CheckpointLivenessUpdater implements ILivenessUpdater {
+    private ScheduledExecutorService executorService;
+    private static final int updateInterval = 15000;
```

Comment:
Please use Duration

Reply:
At the usage it's required in long anyway. But renaming it to UPDATE_INTERVAL_MILLIS  to be explicit about the TimeUnit